### ISSUE TITLE
[A11y] Contrasts partly seem not to be WCAG-compliant (AAA & AA)

### ISSUE THREAD

**Issue by mariohamann** [corona-warn-app/cwa-app-ios#229]
Also described in: https://github.com/corona-warn-app/cwa-app-android/issues/174#issue-631080966

## Current Implementation
I have analyzed the [High-Resolution Screenshots](https://github.com/corona-warn-app/cwa-documentation/blob/master/ui_screens.md) with the [WebAIM Contrast-Checker](https://webaim.org/resources/contrastchecker/) I found that some of the contrasts do not reach WCAG AAA, sometimes not even WCAG AA.

(The following image may show other colours because of web compression. Here the Screenshot of the Android-App is shown, but the colours match those of iOS)
![contrasts](https://user-images.githubusercontent.com/26542182/83799430-12e4fc00-a6a6-11ea-996e-290ae74b17b3.jpg)

Screen 1 (start screen): Both the green area and the blue button do not offer enough contrast. In addition, the perception of the color "blue" decreases most with age ([Source](https://www.smashingmagazine.com/2015/02/designing-digital-technology-for-the-elderly/)), which is why the lower button in combination with the continuous capitalization and the narrow font could cause problems for certain user groups. 

Screen 2 ("Low risk") and 3 ("Risk determination") both contain a small grey font. This is just above the AA minimum of 4.5 from a contrast perspective, but in combination with the very small font could be very difficult to read for some user groups. 

Screen 3 ("Risk assessment"): The switch may not be recognizable as such for some user groups - some may just see a dot.
## Suggested Enhancement
For reasons of accessibility, the contrasts should be increased. At the same time, the difference between headings and "secondary text" must of course be maintained. In iOS-version this is better implemented as some headings are bolder.

## Expected Benefits
The application should be easy to use for all user groups - but individual risk groups such as senior citizens are very likely to have higher usability requirements, also in terms of contrasts. In order not to exclude these user groups from using the application, WCAG AAA should be implemented in the best possible way.

**Comment by tkowark** [comment]
Will be answered together with https://github.com/corona-warn-app/cwa-documentation/issues/195
